binstar_client/utils/__init__.py

```diff
@@ -247,8 +245,8 @@ def recursive_update(d, u):
             d[k] = u[k]
     return d
 
-DEFAULT_URL = 'https://api.anaconda.org'
```

Comment:
In the line that Sean referenced, if 'url' is in the config then it will override the default.

I think you'd want to change that line instead, to something like `os.environ.get('ANACONDA_DEFAULT_SITE', config.get('url', DEFAULT_URL))`